src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.getElementsByClassName('wall')[0].addEventListener('click', e => {
```

Comment:
Better practice is to use `document.querySelector('.wall')`, it picks first element in the DOM with this class. Check other similar cases.

Reply:
ok, thanks